theme.json

```diff
@@ -1,6 +1,25 @@
 {
   "$schema": "https://schemas.wp.org/trunk/theme.json",
   "version": 2,
+  "styles": {
```

Comment:
Maybe we could move the link color changes there as well, but since these are not broken by the upgrade I figured it might be better to do this in a separate PR!
Also, I'm not sure that we would want this since it might make it more difficult for NROs to customise? I don't know if they can use `theme.json` for that 🤔

Reply:
Can you create a follow up ticket with what we can move over to `theme.json`? It seems that's technically possible to ovveride things through its own `theme.json` file ([example](https://wpastra.com/docs/override-astras-theme-json/)). We could document that process.